src/main/java/com/cinema/controllers/MovieSessionController.java

```diff
@@ -0,0 +1,46 @@
+package com.cinema.controllers;
+
+import com.cinema.model.dto.MovieSessionRequestDto;
+import com.cinema.model.dto.MovieSessionResponseDto;
+import com.cinema.service.MovieSessionService;
+import com.cinema.service.impl.mapper.MovieSessionMapper;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema/movie-sessions")
+public class MovieSessionController {
+    private static final String DATE_FORMATTER = "d-MM-yyyy";
+    private MovieSessionService movieSessionService;
+    private MovieSessionMapper movieSessionMapper;
+
+    public MovieSessionController(MovieSessionService movieSessionService,
+                                  MovieSessionMapper movieSessionMapper) {
+        this.movieSessionService = movieSessionService;
+        this.movieSessionMapper = movieSessionMapper;
+    }
+
+    @PostMapping()
+    public void addMovieSession(@RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        movieSessionService.add(movieSessionMapper
+                .mapToMovieSessionFromDto(movieSessionRequestDto));
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> getMovieSessions(@RequestParam Long id,
+                                                          @RequestParam String date) {
```

Comment:
This is optional but consider following approach: 
@RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date

Reply:
ok, will try - had some issues how to test it in Postman before: decided not to stop the rest. Now will try to check it...